lib/index.js

```diff
@@ -20,11 +21,15 @@ let DEFAULTS = {
 		static: {
 			plugin: "faucet-pipeline-static",
 			bucket: "static"
+		},
+		nunjucks: {
+			plugin: "faucet-pipeline-nunjucks",
+			bucket: "markup"
```

Comment:
What's the rationale for adding this here rather than relegating it to project-specific configuration? All the other plugins are pretty much generic WRT the web pillars (admittedly Sass is an opinionated outlier, but see below), so this seems like a case of "one of these is not like the others" to me.

I'm wary of having to release a new version of faucet-core whenever we create new plugins, useful as they might be - let alone removing them again if/when we lose interest (which seems significantly more likely for Nunjucks than for Sass).

If extending `plugins` from within configuration is too awkward, we should address that (e.g. `require("faucet-pipeline-nunjucks/register"); module.exports = …`).

Reply:
I agree that we should not allow arbitrary plugins into the "registry". But I would say that:

1. The "pillars of the Web" is way too strict. This would also exclude Sass, TypeScript & JSX. As HTML doesn't make any sense by itself, we could only keep JS (and ESNext) plus CSS (which is not even done) here. This doesn't seem helpful. I would rephrase that as "things that are common (like SCSS, TypeScript...) and that we use regularly". This would probably make those "official" plugins. 
2. I would argue that the "official" plugins should also be those that are documented on faucet-pipeline.org – all "unofficial" plugins must contain their own documentation. We should however have a list of "unofficial" pipelines (we know of) on that page. It could also be argued that only the official plugins should be in the Github org.
3. Using faucet-pipeline in combination with a template engine like nunjucks is an interesting use case, because it is a very simple static site generator. I would like to demonstrate that use case on faucet-pipeline.org somehow.
4. To make registering the plugins less akward, I could imagine something like this in the faucet.config.js (this would require "faucet-pipeline-foo/register" which is the usual object with a plugin and bucket key):
    ```js
    module.exports = {
      foo: [{ ... }],
      plugins: [
        "faucet-pipeline-foo"
      ]
    }
    ```

To sum it up, I agree to remove that commit. I however would like to have a system of "official" and "unofficial" plugins/pipelines and an easier way for users to register unofficial plugins.